gen-go-build.py

```diff
@@ -36,7 +39,7 @@ def maybe_gen(dir, go_version, arch):
         os.chmod(filename, 0755)
 
 def mac():
-    for mac_version in mac_versions:
+    for mac_version, go_versions in mac_versions:
```

Comment:
Shouldn't this be `for mac_version, go_versions in mac_versions.items()` ?

Reply:
Yeah, it probably should be, I'm not a first class python programmer quite yet.